lib/api/objectDelete.js

```diff
@@ -51,6 +51,13 @@ function objectDeleteInternal(authInfo, request, log, isExpiration, cb) {
     const reqVersionId = decodedVidResult;
     const hasGovernanceBypass = hasGovernanceBypassHeader(request.headers);
 
+    // Reassign a specific apiMethod, as monitoring can rely on the
+    // type of cloudserver instance to determine it comes from Backbeat routes.
+    // This is also needed for quota evaluation, where "routeBackbeat" cannot
+    // be used for different operations, such as putObject
+    // eslint-disable-next-line no-param-reassign
+    request.apiMethod = reqVersionId ? 'objectDeleteVersion' : 'deleteObject';
```

Comment:
would it not be better to keep the existing `apiMethod`, and just pass a different `requestType` in case of `isExpiration` ?
→ this ensures only `valParams` is changed, i.e. only the behavior of `standardMetadataValidateBucketAndObj()` is affected
→ it explicitely tweaks the behavior for the backbeat route only, ensuring the "user" api/path is not affected